crates/feldera-cli/README.md

```diff
@@ -0,0 +1,9 @@
+# Feldera CLI
+
+A CLI tool for interacting with the Feldera REST API.
+
+```commandline
+cd crates/feldera-cli
+cargo install --path .
+feldera --help
```

Comment:
Better to called it fctl like we do for the cli in the cloud repo (everything getting called Feldera doesn't help).

Reply:
but thats horrible we dont want to end up with two different binaries of the same name in the same path? what about feld? we should the  then rename the pipeline-manager to feldera-* so we can eventually publish it on crates.io/feldera

Reply:
I don't see much value to publishing pipeline-manager to crates.io. It is unusable without the rest of the build, which is quite bespoke (java, typescript, build.rs etc).

Reply:
But yes, having a single cli would be pretty useful. Worth coordinating with @snkas

Reply:
I think that can be fixed... typescript inclusion already works as is today with crates.io its just about building the jar on the fly.

Reply:
I think this client-facing one interacting through the API is the more important one, I can rename the other CLI tool.

Reply:
I think this client-facing one is the more important one, I can rename the other CLI tool.